include/ProgramRepresentation/PVAliasSet.h

```diff
@@ -0,0 +1,91 @@
+#ifndef PV_ALIAS_SET_H
+#define PV_ALIAS_SET_H
+
+#include "ProgramRepresentation/MethodsSet.h"
+#include "ProgramRepresentation/ProgramVariable.h"
+#include "Utils.h"
+
+// holds a set of program variables and methods called on them.
+// each program variable in the set is a must-alias of all the other variables in that set.
+class PVAliasSet {
+  private:
+    std::list<ProgramVariable> programVariables;
+
+    // holds the methods called on the set
+    MethodsSet methods;
+
+  public:
+    PVAliasSet();
+
+    // returns true iff programVar is in the set by checking its raw name
```

Comment:
"by checking its raw name" is an implementation detail. In general, it's better to say "what" a method does in a description like this, but leave "how" ambiguous. This method is the preferred way to check if a ProgramVariable is in the set, so it should say only that.